ruby-engine/spec/yggdrasil_engine_spec.rb

```diff
@@ -25,6 +25,22 @@ def test_suite_variant(base_variant)
     end
   end
 
+  describe '#getting state' do
+    it 'should return JSON and support roundtrip' do
+       empty_state = yggdrasil_engine.get_state
+       expect(empty_state).to include('"features":[]')
+
+       test_state = '{"version":1,"features":[{"name":"testFeature","enabled":true,"strategies":[{"name":"default"}]}]}'
+       yggdrasil_engine.take_state(test_state)
+       retrieved_state = yggdrasil_engine.get_state
+
+       expect(retrieved_state).to include('testFeature')
```

Comment:
Ah that makes sense. These tests are burning us here because the response contains the wrapping so it's not correct but does these individual properties.

Damnit, that's quite an easy and subtle trap to fall into. Would love your thoughts on how we can make this a harder thing to do by mistake